Providers/Modules/Plugins/PatchManagement/plugin/filter_patch_management.rb

```diff
@@ -30,6 +30,14 @@ def shutdown
     end
 
     def filter(tag, time, record)
+      begin
+        File.delete("/tmp/az-update-security.list")
+      rescue Exception => e
```

Comment:
can you update corresponding dsc module version since this is functional change?

Reply:
yes will do